src/panels/mailbox/ha-panel-mailbox.ts

```diff
@@ -19,6 +19,7 @@ import { HomeAssistant } from "../../types";
 import { fireEvent } from "../../common/dom/fire_event";
 import { haStyle } from "../../resources/styles";
 import "../../components/ha-top-app-bar-fixed";
+import "../../resources/intl-polyfill";
```

Comment:
To prevent having to manage this everywhere, we should just move the format function to a file that imports this.